### DISCUSSION TITLE
could we add ConnectConfiguration with http's DisableKeepAlives option?

### DISCUSSION THREAD

**Discussion by baiweiguo** [beltran/gohive#197] [Ideas]
![image](https://user-images.githubusercontent.com/3157054/188870310-69ac5b31-e3ac-4288-baaf-61ac5fc860a6.png)

**Comment by beltran** [comment]
Yes, I don't see a problem with this.

**Comment by beltran** [comment]
This was added [here](https://github.com/beltran/gohive/commit/e5031800801534fe8947a8fa2b3382bd23952285).

**Comment by baiweiguo** [comment]
您好！您的邮件已收到，我将会尽快地进行处理，谢谢来信！
